config/settings.yml

```diff
@@ -4,6 +4,23 @@
     :blacklisted_event_names: []
     :event_handling:
       :event_groups:
+        :addition:
+          :critical_regex:
+          - ^nuage_.+_create$
+        :deletion:
+          :critical_regex:
+          - ^nuage_.+_delete$
+        :update:
+          :critical_regex:
+          - ^nuage_.+_update$
+          :detail:
+          - nuage_vsc_update
+          - nuage_vrs_update
+        :status: # alarms
+          :critical:
```

Comment:
This has changed a little, please remove the `:critical:` category and simply use

```yaml
:critical_regex:
- ^nuage_alarm_.*$
```

Reply:
Consider it done.